### DISCUSSION TITLE
Feature Request: Swap Right and Left Layout for Mobile

### DISCUSSION THREAD

**Discussion by seenanadimi** [puckeditor/puck#664] [Q&A]
Currently, the layout on mobile devices displays the right and left sections in the same order as the desktop view. However, I think it would be beneficial for the layout to adapt based on the device being used, with the right section appearing first on mobile (if applicable). This would enhance the readability of the outline for different layouts.

Could this be considered for future development? I'd be happy to discuss or help clarify any details if needed.

Thanks again for your time and for considering this feature request!

**Comment by chrisvxd** [comment]
Heya @seenanadimi! Sorry for slow response. I've been missing GH discussions recently.

Are you suggesting that the right sidebar is opened before the left sidebar on smaller viewports, or that the contents of sidebars is swapped completely?

FWIW, happy for feature requests to be opened as issues.
